testing/testHttpServer.go

```diff
@@ -172,8 +173,9 @@ func (ths *TestHttpServer) processRequestGetConfigMetrics(rw http.ResponseWriter
 		"erd_round_duration":             30,
 		"erd_shard_consensus_group_size": 30,
 		"erd_start_time":                 30,
+		"erd_min_transaction_version":    1,
 	}
-	resp := data.GenericAPIResponse{Data: responseStatus, Code: data.ReturnCodeSuccess}
+	resp := data.GenericAPIResponse{Data: gin.H{"config": responseStatus}, Code: data.ReturnCodeSuccess}
```

Comment:
why you use gin.H here ?

Reply:
as to map the `data` field to `key: value`. the way it was before didn't work and reading network configuration failed.